### PR TITLE
To Do list: interactive list

### PR DESCRIPTION
**Pull request by anas599**
- Add a new JavaScript file and import it as a module:
- Add methods related to the status updates (completed: true/false).
- Update items object's value for completed key upon user actions.
- Implement a function for the "Clear all completed" button

### CODE REVIEW

File: webpack-demo/src/listFunctions.js [hunk 1]
```diff
@@ -26,7 +27,15 @@ function displayTasks() {
     const checkbox = document.createElement('input');
     checkbox.type = 'checkbox';
     checkbox.id = 'checkbox';
+    if (toDoArr.completed) {
+      checkbox.setAttribute('checked', true);
+      li.classList.add('completed');
+    }
     li.appendChild(checkbox);
+    checkbox.addEventListener('click', () => {
+      // eslint-disable-next-line no-use-before-define
```
Comment (elisha2kyakpo1) [inline webpack-demo/src/listFunctions.js:36]: - [x] Let's not disable linters because they are there to correct errors and make our code follow standards.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @anas599, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description look good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
✔️ The app functionality works as expected👏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: webpack-demo/src/listFunctions.js (hunk 1 above)
Comment (anas599) [reply webpack-demo/src/listFunctions.js:36]: Hi @elisha2kyakpo1  thanks for pointing it out
can you help me out or suggest what to do to solve it?

Comment (DeliceLydia) [reply webpack-demo/src/listFunctions.js:36]: Addressed

**Review by DeliceLydia — APPROVED** [review]
## Hi @anas599, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**